### DISCUSSION TITLE
How can I serve things in a hidden directory?

### DISCUSSION THREAD

**Discussion by jbanana** [mbrubeck/agate#151] [Q&A]
I created a `security.txt` file for my capsule. It's at the path `/.well-known/security.txt` (where it's supposed to be) but notice the dot. I get a 52 response if I fetch it. I can't figure out how to allow the request with `.meta` directives. Is it possible?

**Comment by Johann150** [comment]
The readme says:
> If any segment of the requested path starts with a dot, agate will respond with a status code 52, whether the file exists or not. This behaviour can be disabled with `--serve-secret` or by an entry for the specific file in the `.meta` configuration file.

It's right at the top of the file. Do you think something could be changed to make it more discoverable?

**Reply by jbanana** [reply]
I saw that in the documentation - it was very discoverable!

- I don't want all dot files to be visible, so `--serve-secret` doesn't helping me.
- I tried making a `.meta` entry, but the hidden thing is a directory, not a file to be served, so it didn't work.

In the root-level `.meta` file I added
`.well-known: `
but I don't know what to put as the metadata part. None of the 4 options seems to apply to a directory. If I put this line
`.well-known/security.txt: text/plain`
it is ignored because the file I want to serve isn't in the directory - it's in the one below. I'm not using `--central-conf`.

Apologies if I'm missing something. What should I put in the `.meta` file?

**Comment by Johann150** [comment]
> it is ignored because the file I want to serve isn't in the directory - it's in the one below.
I don't quite understand that part.

As I explained in an earlier discussion the patterns are "compiled" so it's not possible to have them apply to a directory. You could try something that applies to all files in the directory with a glob like .well-known/*
